docs/extensions-plugins/obsidian.mdx

```diff
@@ -0,0 +1,186 @@
+  return (
+    <img
+      style={{ marginLeft: "0px", height: "30%", marginRight: "12px" }}
+      src={"/assets/obsidian.png"}
+    />
+  );
+};
+
+<GridFlexRow>
+  <h1>Pieces Obsidian Plugin</h1>
+</GridFlexRow>
+
+<br></br>
+
+<GridFlexRow>
+  <InstallButton>Get the Obsidian Plugin</InstallButton>
+  <ReleasePill children="Current Version: {{ versions.obsidian }}"></ReleasePill>
+</GridFlexRow>
+
+<br></br>
+
+# Your Guide to Getting Started with Pieces for Obsidian
+
+_Pieces for Obsidian Pieces captures **NO identifiable user data**. Our **Local only** architecture means your notes never have to leave your device!_
+
+_The <a target="_blank" href="{{ links.website.pfd_desktop_install }}">Pieces Suite</a> must be downloaded and installed separately from the Obsidian Plugin in order for the Plugin to work properly._
+
+## Save your Snippets
+
+- To save a snippet, highlight the text, right-click, and select "Save to Pieces."
+  **OR**
+- Click the Pieces button within any code block.
+  **OR**
+- Highlight the desired code and use our dedicated keyboard shortcut
+  `Ctrl+Shift+P` (\[⌘⇧P\] on macOS)
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_RIGHTCLICK.gif" />
+
+## Find your Snippets
+
+Open the Pieces View from the Ribbon Menu.
+
+- View by **Recent** and by **Language**
+- Search your snippets to save time searching through your markdown.
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_SEARCH.gif" />
+
+## Use your Snippets
+
+- Copy and share your snippets from the Pieces View
+- You can expand your code snippets to view more meta-data that Pieces automatically generates for you!
+- View your snippets with syntax highlighting
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_VIEWPIECE.gif" />
+
+## Share your Snippets
+
+- To share a snippet, highlight the text, right-click, and select "Share via Pieces Link."
+  **OR**
+- Click the Pieces Share button within any code block / in the Pieces View.
+  **OR**
+- Highlight the desired code and use our dedicated keyboard shortcut
+  `Ctrl+Shift+L` (\[⌘⇧L\] on macOS)
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_SHARE.gif" />
+
+_Pieces offers hosting for all your snippets for seamless sharing!_
+
+## Embedded Buttons
+
+Pieces adds **Embedded Buttons** into your markdown to help you save, copy, & share your code snippets.
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_SAVE.gif" />
+
+## Having Trouble with Pieces for Obsidian?
+
+- To double-check that Pieces OS is running, make sure that the Pieces logo is present in your toolbar (on macOS) or your task bar (on Windows). If it isn't there, please launch Pieces OS by clicking Pieces OS in your Applications folder.
+- If you need to download and install Pieces OS, please <a target="_blank" href="https://docs.pieces.app/overview">visit this link</a>.
+
+If the Obsidian plugin still isn't working for you, please make sure you have the following:
+
+- The latest version of Pieces for Obsidian
+- Pieces OS version 5.0.0 or higher
```

Comment:
there is not a version for this yet stored as a variable but would be a good opportunity, im going to create an issue for this